backends/platform/psp/default_display_client.cpp

```diff
@@ -176,8 +176,7 @@ void Screen::setScummvmPixelFormat(const Graphics::PixelFormat *format) {
 	PSP_DEBUG_PRINT("format[%p], _buffer[%p], _palette[%p]\n", format, &_buffer, &_palette);
 
 	if (!format) {
-		bzero(&_pixelFormat, sizeof(_pixelFormat));
-		_pixelFormat.bytesPerPixel = 1;	// default
+		_pixelFormat = Graphics::PixelFormat::createFormatCLUT8();	// default
```

Comment:
This change seems to make sense to me, but results in a change in behaviour (with the PixelFormat `rLoss`, `gLoss`, `bLoss`, and `aLoss` now having a value of 8 instead of 0). This was probably a bug, and maybe doesn't even matter for indexed colors in most cases, but at least the result of `_pixelFormat == Graphics::PixelFormat::createFormatCLUT8()` will be different. So maybe this should be double checked by somebody familiar with the PSP backend.